components/Empty.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "../styles/components/Empty.module.css";
+import React from "react";
+
+// Displayed when there is no data to display
+const Empty: React.FC = () => {
+  return <div className={styles.empty}>There are no Staff.</div>;
```

Comment:
hmm. Probably don't want to define this as a component with Staff hardcoded in. How about it takes in a prop that can either be "Classes" | "Staff" | "Students" and the message can be "There are no {"Classes" | "Staff" | "Students"} to display" or something like that. 

Also, can we move the Error, Loader, and Empty components into a `components/util` directory.